doc/modules/preprocessing.rst

```diff
@@ -593,6 +593,51 @@ the 2 features with respectively 3 categories each.
 See :ref:`dict_feature_extraction` for categorical features that are
 represented as a dict, not as scalars.
 
+.. _target_regressor_encoder:
+
+Target Regressor Encoder
+========================
+The :class:`~sklearn.preprocessing.TargetRegressorEncoder` uses statistics of
+the target conditioned on the categorical feature. The target encoding scheme
+takes a weighted average of the overall target mean and the target mean
+conditioned on categories. A multilevel linear model, where the levels are the
+categories, is used to construct the weighted average is estimated in Chapter
```

Comment:
They call it pooled average, right? Maybe say that? Also maybe provide the other references, even though not entirely applicable?

Reply:
Isn't it just "multilevel linear model", i.e. not "generalized"? Or did I miss something essential?

Just as info: A multilevel linear model with only a random intercept, i.e. the approach chosen here, is equivalent/aka (Bühlmann Straub) "credibility estimator". Maybe too much information for the UG.